apps/landingPage/src/pages/index.js

```diff
@@ -3,15 +3,18 @@
 import React from 'react';
 import styled, { createGlobalStyle } from 'styled-components';
 
-import Header from '../components/header';
-import Footer from '../components/footer';
-import Demo from '../components/demo';
-import Info from '../components/info';
-import Headline from '../components/headline';
-import Playground from '../components/playground';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import Demo from '../components/Demo';
+import Info from '../components/Info';
+import Headline from '../components/Headline';
+import Playground from '../components/Playground';
 
-const iframeWidth = 1200;
-const iframeHeight = 800;
+const windowWidth = window.innerWidth; //eslint-disable-line
+const windowHeight = window.innerHeight; //eslint-disable-line
```

Comment:
I think that we can extend our the eslint setting about ` "browser": true`.

Reply:
Ok! added "browser": true there